config/mainCfg_ETau_UL2016.cfg

```diff
@@ -0,0 +1,132 @@
+[general]
+
+lumi = 16800 # pb^-1 2016 postVFP
+
+outputFolder = analysis_ETau_UL2016_13Apr2022_newPostVFP
+
+data = DsingleEleF, DsingleEleG, DsingleEleH
+#data = DsingleEleF, DsingleEleG, DsingleEleH, DMETF, DMETG, DMETH
+
+signals = ggHTauTau
+
+backgrounds = TTfullyHad, TTfullyLep, TTsemiLep, DY, DY_Incl, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, WW, WZ, ZZ, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB, WWW, WWZ, WZZ, ZZZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu, TTWW, TTWZ, TTZZ
+
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, bjet1_eta, bjet2_eta, tauH_mass, tauH_SVFIT_mass, tauH_pt, tauH_SVFIT_pt, ditau_deltaR, bH_pt, bH_mass, HH_pt, HH_mass, HHKin_mass, met_et, met_phi
+
+#selections = baseline, s1b1jresolvedMcut, s2b0jresolvedMcut, sboostedLLMcut
+selections = baseline
```

Comment:
It could be interesting to include the 1b1j 2b0j and boosted categories (here and in the other channels), with inverted ellyptical mass cut.